altos-rust/port/cortex-m0/src/peripheral/usart/mod.rs

```diff
@@ -0,0 +1,126 @@
+// peripheral/serial/mod.rs
+// AltOSRust
+//
+// Created by Daniel Seitz on 11/30/16
+mod control;
+mod defs;
+mod baudr;
+
+use super::Control;
+use volatile::Volatile;
+use self::control::UsartCR;
+use self::baudr::UsartBRR;
+use self::defs::*;
+use peripheral::rcc;
+
+pub use self::control::{WordLength, Mode, Parity, Stoplength, HardwareFlowControl};
+pub use self::baudr::BaudRate;
+
+#[derive(Copy, Clone)]
+pub enum USARTx {
```

Comment:
`UsartX`, and make the variants `Usart1` and `Usart2` so the casing is consistent